.github/ISSUE_TEMPLATE/new_snippet.md

```diff
@@ -0,0 +1,22 @@
+---
+name: New snippet
+about: New snippet template
+title: 'New snippet: ...'
+labels: 'new snippets'
+---
+
+<!--
+Hi, thanks for submitting a new snippet. We appreciate that.
+
+But, we will need some information and explanation to accept it.
+-->
+## Description
+
+## Snippet preview
+
+## Checklist before calling for maintainers
+
+* [ ] Have you checked to ensure there aren't other open [Issues](../../../issues) for the same update/change?
+* [ ] Have you checked that this snippet is not similar to any of the existing snippets?
+* [ ] Have you added an `Explanation` section? It shall include the reasons of changes and why you'd like us to include them
```

Comment:
let's mark the explanation section optional, sometime people come to us without having exact explanation